source/backend/api/Areas/Property/Controllers/PropertyController.cs

```diff
@@ -62,6 +65,23 @@ public IActionResult GetPropertyAssociationsWithId(long id)
 
             return new JsonResult(_mapper.Map<PropertyAssociationModel>(property));
         }
+
+        /// <summary>
+        /// Get all associated leases for the specified unique property 'id'.
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet("{id}/leases")]
+        [HasPermission(Permissions.PropertyView)]
+        [HasPermission(Permissions.LeaseView)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(IEnumerable<Api.Models.Concepts.PropertyLeaseModel>), 200)]
+        [SwaggerOperation(Tags = new[] { "property" })]
+        public IActionResult GetPropertyLeases(long propertyId)
```

Comment:
I think we need to confirm that this approach is appropriate given that is requires LeaseView. That means that a user that has management view will get an error unless they also have lease-view when viewing the management summary.

We may need to return just the boolean field against the property calculated on the backend with no lease restrictions instead.

Reply:
@asanchezr

Reply:
I see, as said in the PR description this is the basic model stuff. I can change that in my next PR